pkg/reconciler/pipelinerun/tracing.go

```diff
@@ -65,7 +65,7 @@ func initTracing(ctx context.Context, tracerProvider trace.TracerProvider, pr *v
 	// Create a new root span since there was no parent spanContext provided through annotations
 	ctxWithTrace, span := tracerProvider.Tracer(TracerName).Start(ctx, "PipelineRun:Reconciler")
 	defer span.End()
-	span.SetAttributes(attribute.String("taskrun", pr.Name), attribute.String("namespace", pr.Namespace))
+	span.SetAttributes(attribute.String("pipelinerun", pr.Name), attribute.String("namespace", pr.Namespace))
```

Comment:
Should I change this to a const value? We might need to change this in Taskrun reconciler also if I do that.

Reply:
I think this is ok. I'm not opposed to add a const though.

Reply:
you could use the constants from this file https://github.com/tektoncd/pipeline/blob/main/pkg/apis/pipeline/controller.go

Reply:
@lbernick These start with caps. The convention is to have it all in lowercase.